### PR TITLE
dev-lua/luaexpat: new maintainer

### PR DESCRIPTION
**Pull request by winterheart**
Added myself to maintainers list, removed old version, switch to EAPI 7,
fixes for HOMEPAGE.
Closes: https://bugs.gentoo.org/730342

Package-Manager: Portage-2.3.79, Repoman-2.3.16
Signed-off-by: Azamat H. Hackimov <azamat.hackimov@gmail.com>

### CODE REVIEW

File: dev-lua/luaexpat/luaexpat-1.3.0-r2.ebuild [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs flag-o-matic eutils multilib-minimal
+
+DESCRIPTION="LuaExpat is a SAX XML parser based on the Expat library"
+HOMEPAGE="http://matthewwild.co.uk/projects/luaexpat/"
+SRC_URI="http://matthewwild.co.uk/projects/${PN}/${P}.tar.gz"
```
Comment (juippis) [inline dev-lua/luaexpat/luaexpat-1.3.0-r2.ebuild:10]: `https://` seems to be available.

File: dev-lua/luaexpat/luaexpat-1.3.0-r2.ebuild [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~hppa ~ppc ~ppc64 ~sparc ~x86"
+IUSE=""
```
Comment (juippis) [inline dev-lua/luaexpat/luaexpat-1.3.0-r2.ebuild:15]: These empty variables can be dropped, if you wish.

File: dev-lua/luaexpat/luaexpat-1.3.0-r2.ebuild [hunk 3]
```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs flag-o-matic eutils multilib-minimal
```
Comment (juippis) [inline dev-lua/luaexpat/luaexpat-1.3.0-r2.ebuild:6]: It doesn't look like *eutils.eclass* is being used in here.

**Comment by winterheart** [comment]
@juippis done

File: dev-lua/luaexpat/luaexpat-1.3.3.ebuild [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+src_prepare() {
+	default
+	multilib_copy_sources
+	append-flags -fPIC
+}
+
+multilib_src_compile() {
+	emake \
+		CFLAGS="${CFLAGS}" \
+		LDFLAGS="${LDFLAGS}" \
```
Comment (juippis) [inline dev-lua/luaexpat/luaexpat-1.3.3.ebuild:30]:
```
 * Messages for package dev-lua/luaexpat-1.3.3:

 * QA Notice: Files built without respecting LDFLAGS have been detected
 *  Please include the following list of files in your report:
 * /usr/lib/lua/5.1/lxp.so.1.3.3
 * /usr/lib64/lua/5.1/lxp.so.1.3.3
```

Maybe this doesn't work, or these files are special?

Comment (winterheart) [reply dev-lua/luaexpat/luaexpat-1.3.3.ebuild:33]: Fixed by patching makefile.

**Comment by winterheart** [comment]
@juippis fixed.

**Review by juippis — APPROVED** [review]

**Pull request closed without merging**